gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/inmemory/MembershipQueryServiceInMemory.java

```diff
@@ -62,6 +62,18 @@ public Collection<Membership> findByReferencesAndRoleId(
             .toList();
     }
 
+    @Override
+    public Collection<Membership> findGroupsUserBelong(String userId) {
+        return storage
+            .stream()
+            .filter(membership ->
+                membership.getReferenceType().equals(Membership.ReferenceType.GROUP) &&
+                membership.getMemberType().equals(Membership.Type.USER) &&
+                membership.getMemberId().equals(userId)
```

Comment:
do you think this predicate can be written as a method of membership object ?
like `membership.isGroupUpser(String userId)`

Reply:
I like the idea 💡